docs/changelog.md

```diff
@@ -1,5 +1,10 @@
 # Changelog
 
+## 1.3.0
```

Comment:
This needs to be updated to 1.4.0, since 1.3.0 now already exists

Reply:
Good call-out! Will adjust before merging